templates/components/_header.html

```diff
@@ -26,7 +26,7 @@
 								<li class="menu-item-has-children">
 									<a href="javascript:void(0);">More</a>
 									<ul class="sub-menu">
-										<li><a href="/#speakers">Speakers</a></li>
+										<li><a href="#speakers">Speakers</a></li>
```

Comment:
@thisisayush how about someone tries to visit the speakers section from some other page, for example, say from sponsorship.html? I think this will be wrong in that. Because staticjinja will render it for all the pages. Re-check.